packages/gsm8k/src/aviary/envs/gsm8k/env.py

```diff
@@ -145,28 +145,27 @@ def submit_answer(self, answer: str) -> tuple[bool, float, Literal[True]]:
         else:
             return correct, reward, True
 
-    def calculator(self, expr: str) -> tuple[float | str | None, float, bool]:
+    def calculator(self, expr: str) -> tuple[float | str, float, bool]:
         """Calculate a mathematical expression.
 
         Args:
-            expr (str): A valid python expression
+            expr: A valid Python expression.
 
         Returns:
-            float: Result of the expression
+            A three-tuple where the first element is the float evaluation if successful,
+                or a string containing the failure cause if unsuccessful, the second
+                element is the reward associated with success or failure, and the third
+                element is a boolean indicating if this action is terminal.
```

Comment:
Let's not do docstring changes right now, since this may affect performance and require rerunning experiments.